clients/tabby-chat-panel/src/index.ts

```diff
@@ -1,15 +1,15 @@
-import { MessageEndpoint, createEndpoint, fromIframe, fromInsideIframe } from '@remote-ui/rpc'
+import type { Thread, ThreadOptions } from '@quilted/threads'
 
 export interface LineRange {
   start: number
-  end: number
+  end?: number
 }
 
 export interface FileContext {
   kind: 'file'
-  range: LineRange
-  filename: string
-  link: string
+  range?: LineRange
```

Comment:
```suggestion
  range: LineRange
```

Range should be required.